### PR TITLE
Try decrypting config

### PR DESCRIPTION
**Pull request by alisterscott**
Uses the new encrypted config values for live branch execution

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ test:
   pre:
 
   override:
+    - echo decrypting config
+    - cd wp-e2e-tests && npm run decryptconfig
     - echo running test command "./run.sh -R -C $RUN_ARGS"
     - cd wp-e2e-tests && git checkout origin/${E2E_BRANCH-master} && ./run.sh -R -C $RUN_ARGS
```
Comment (brbrr) [inline circle.yml]: I think we should run `decryptconfig` after checkout. For testing you can try set `NODE_CONFIG={}` when you run the tests: `&& NODE_CONFIG={} ./run.sh -R -C $RUN_ARGS`

**Comment by alisterscott** [comment]
> I think we should run decryptconfig after checkout

@brbrr thanks, I've moved it and tested it out using an empty NODE_CONFIG and it appears to work fine

@hoverduck I think you were the one who added the checkout where it is. Is there a reason it isn't in the checkout section, as wouldn't we not get the dependencies correct if we're checking out a different branch?

File: circle.yml (hunk 1 above)
Comment (alisterscott) [reply circle.yml]: Thanks, fixed

**Comment by brbrr** [comment]
🐑 🇮🇹

**Pull request merged**